persistence-modules/java-jpa-2/src/test/java/com/baeldung/jpa/multipletables/MultipleTablesIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+package com.baeldung.jpa.multipletables;
+
+import static org.assertj.core.api.Assertions.*;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.baeldung.jpa.multipletables.multipleentities.MealWithMultipleEntities;
+import com.baeldung.jpa.multipletables.secondarytable.MealAsSingleEntity;
+import com.baeldung.jpa.multipletables.secondarytable.embeddable.MealWithEmbeddedAllergens;
+
+public class MultipleTablesIntegrationTest {
+
+    private static EntityManagerFactory emf;
+    private static EntityManager em;
+
+    @BeforeClass
+    public static void setup() {
+        emf = Persistence.createEntityManagerFactory("jpa-h2-multipltables");
+        em = emf.createEntityManager();
+    }
+
```

Comment:
Could we use test names as per the author documentation like given_xxx_. Thanks.

Reply:
I am aware of the naming convention. However, we don't have anything given in my opinion, that's why I got rid of that part. Also, the documentation allows it: "given part is optional, but the other two are not"

Should I still rename them?